hknweb/init_permissions.py

```diff
@@ -3,42 +3,35 @@
 from django.contrib.auth.models import User, Group, Permission
 from django.contrib.contenttypes.models import ContentType
 
-candidate = Group.objects.get_or_create(name='candidate')[0]
-officer = Group.objects.get_or_create(name='officer')[0]
+candidate, _created = Group.objects.get_or_create(name='candidate')
+officer, _created = Group.objects.get_or_create(name='officer')
 
 #CANDIDATE AND OFFICER PERMISSIONS
-announcement = ('view_announcement', Announcement)
-add_offchallenge = ('add_offchallenge', OffChallenge)
-view_offchallenge =	('view_offchallenge', OffChallenge)
-change_offchallenge = ('change_offchallenge', OffChallenge)
-add_bitbyteactivity = ('add_bitbyteactivity', BitByteActivity)
-view_event = ('view_event', Event)
-add_rsvp = ('add_rsvp',Rsvp)
-remove_rsvp = ('delete_rsvp',Rsvp)
-view_user = ('view_user', User)
-
-cand_permission_names = [announcement, add_offchallenge, view_offchallenge, \
-	change_offchallenge, add_bitbyteactivity, add_rsvp, remove_rsvp, view_user]
+cand_permission_names = [
+    ('view_announcement', Announcement), 
+    ('add_offchallenge', OffChallenge), 
+    ('view_offchallenge', OffChallenge), 
+    ('change_offchallenge', OffChallenge), 
+    ('add_bitbyteactivity', BitByteActivity), 
+    ('add_rsvp',Rsvp), 
+    ('delete_rsvp',Rsvp), 
+    ('view_user', User)
+]
 
 # OFFICER ONLY PERMISSIONS
-add_event = ('add_event', Event)
-change_event = ('change_event', Event)
-
-off_permission_names = [add_event, change_event]
+off_permission_names = [
+    ('add_event', Event), 
+    ('change_event', Event)
+]
 
 # SETTING PERMISSIONS TO GROUPS
-cand_permissions = []
 for perm in cand_permission_names:
```

Comment:
You can use the same tuple destructuring trick here when iterating through a list of tuples:
```suggestion
for codename, model in cand_permission_names:
```